variables.tf

```diff
@@ -144,7 +147,12 @@ variable "default_service_account" {
 
 variable "disable_dependent_services" {
   description = "Whether services that are enabled and which depend on this service should also be disabled when this service is destroyed."
-  default     = "true"
-  type        = string
```

Comment:
Not related to your change, please revert

Reply:
reverted